utils/pylode.py

```diff
@@ -17,9 +17,15 @@ def compute(
     ) -> TensorMap:
         # Step 1: compute spherical expansion with pylode
         hypers = copy.deepcopy(self._hypers)
-        global_species = list(
-            map(int, np.unique(np.concatenate([f.numbers for f in frames])))
-        )
+        
+        if isinstance(frames,list):
+            global_species = list(
+                map(int, np.unique(np.concatenate([f.numbers for f in frames])))
+            )
+        else:
+            global_species = list(
+                map(int, np.unique(np.concatenate([f.numbers for f in [frames]])))
+            )
```

Comment:
Looks good!

Looks good. I just have a minor improvement: You could spare most of the code by just overwriting the frames variable. Something like

```python
if not isinstance(frames, list):
    frames = [frames]
```

With this everything else can be the same as in the current code.

Reply:
Good idea! I just updated the file.